docs/pages/admin-guides/teleport-policy/integrations/azure-sync.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Discover Azure Access Patterns with Teleport Policy
+description: Describes how to import and visualize Azure subscription access patterns using Teleport Policy and Access Graph.
+---
+
+Teleport Policy streamlines and centralizes access management across your entire infrastructure. You can view access
+relationships in seconds, viewing unified, up-to-date relationships and policies between all users, groups, and
+computing resources.
+
+Teleport Policy with Access Graph offers insights into access patterns within your Azure subscription. By scanning
+users, groups, service principals, and role definitions, it provides a visual representation and aids in enhancing the permission model
+within your Azure environment. This functionality enables you to address queries such as:
+
+- What resources are accessible to Azure users and roles?
+- Which groups belong to other groups and how does that impact resource access?
+- How broad or narrow is scoping across the subscription when assigning roles to users and groups?
+
+Utilizing the Access Graph to analyze permissions within an Azure subscription necessitates the setup of the Access
+Graph (AG) service, a Discovery Service, and integration with your Azure subscription.
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product that is available to Teleport Enterprise customers.
+
+After logging in to the Teleport UI, go to the Management tab. If enabled, Access Graph options can be found under
+the Permission Management section.
+
+## How it works
+
+Access Graph discovers Azure access patterns, synchronizes various Azure resources,
+including users, groups, service principals, and role definitions.
+These resources are then visualized using the graph representation detailed in the
+[Teleport Policy usage page](../policy-how-to-use.mdx).
+
+The importing process involves two primary steps:
+
+### Polling Cloud APIs
+
+The Teleport Discovery Service continuously scans the configured Azure subscriptions. At configurable intervals as
+low as 15 minutes, it retrieves the following resources from your Azure account:
+
+- Users
+- Groups
+- Service Principals
+- Role Definitions
+- Role Assignments
+- Virtual Machines
+
+Once all the necessary resources are fetched, the Teleport Discovery Service pushes them to the Access Graph, ensuring
+that it remains updated with the latest information from your Azure environment.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources retrieved from your Azure subscription, crafting a
+graphical representation thereof.
```

Comment:
```suggestion
Teleport Policy’s Access Graph feature visualizes the resources retrieved from your Azure subscription.
```
Simplifying this sentence. We could think about removing the H3-level headings from "How it works", since this H3-level section is so short.